machine/machine_core/constants.py

```diff
@@ -3,8 +3,8 @@
 from lib.constants import OSTREE_IMAGES, MACHINE_DIR, BOTS_DIR, BASE_DIR, TEST_DIR, GIT_DIR, \
     IMAGES_DIR, SCRIPTS_DIR, DEFAULT_IDENTITY_FILE, TEST_OS_DEFAULT, DEFAULT_IMAGE
 
-__all__ = (OSTREE_IMAGES, BOTS_DIR, MACHINE_DIR, BASE_DIR, TEST_DIR, GIT_DIR,
-           IMAGES_DIR, SCRIPTS_DIR, DEFAULT_IDENTITY_FILE, TEST_OS_DEFAULT, DEFAULT_IMAGE,)
+__all__ = ("OSTREE_IMAGES", "BOTS_DIR", "MACHINE_DIR", "BASE_DIR", "TEST_DIR", "GIT_DIR",
```

Comment:
TIL! TBH I'm not sure what `__all__` is even good for. `import *` is just evil. Anyway, thanks for catching this!

Reply:
Yes, it's only for star imports and bad. Maybe we should drop it.